dietpi/dietpi-software

```diff
@@ -8385,6 +8414,118 @@ _EOF_
 			mkdir -p /etc/exports.d
 			[[ -f '/etc/exports.d/dietpi.exports' ]] || echo '/mnt/dietpi_userdata *(rw,async,no_root_squash,fsid=0,crossmnt,no_subtree_check)' > /etc/exports.d/dietpi.exports
 
+		fi 
+    
+		software_id=182 # Unbound
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+		
+		Banner_Configuration
+
+		# Download list of root hints
+		Download_Install https://www.internic.net/domain/named.root /var/lib/unbound/root.hints
+
+		cat << _EOF_ > /etc/unbound/unbound.conf.d/dietpi.conf
+server:
+    verbosity: 0
+    
+    interface: 0.0.0.0
+    port: 53
+    do-ip4: yes
+    do-udp: yes
+    do-tcp: yes
+    
+    do-ip6: yes
+
+    prefer-ip6: no
+
+    root-hints: "/var/lib/unbound/root.hints"
+
+    harden-glue: yes
+    harden-large-queries: yes
+
+    harden-dnssec-stripped: yes
+
+    use-caps-for-id: yes
+
+    edns-buffer-size: 1472
+
+    rrset-roundrobin: yes
+
+    cache-min-ttl: 300
+    cache-max-ttl: 86400
+
+    serve-expired: yes
+
+    harden-algo-downgrade: yes
+
+    harden-short-bufsize: yes
+
+    hide-identity: yes
+
+    identity: "Server"
+
+    hide-version: yes
+
+    do-daemonize: no
+
+    neg-cache-size: 4M
+
+    qname-minimisation: yes
+
+    minimal-responses: yes
+
+    prefetch: yes
+    prefetch-key: yes
+
+    num-threads: 1
+
+    msg-cache-size: 50m
+    rrset-cache-size: 100m
+
+    so-reuseport: yes
+
+    so-rcvbuf: 4m
+    so-sndbuf: 4m
+
+    unwanted-reply-threshold: 10000
+
+    ratelimit: 1000
+
+    log-queries: no
+    log-replies: no
```

Comment:
It seems that this is not compatible with the version shipped on Debian Stretch:
```
Nov 06 21:35:09 VM-Stretch unbound[311]: /etc/unbound/unbound.conf.d/dietpi.conf:68: error: unknown keyword 'log-replies'
Nov 06 21:35:09 VM-Stretch unbound[311]: /etc/unbound/unbound.conf.d/dietpi.conf:68: error: stray ':'
Nov 06 21:35:09 VM-Stretch unbound[311]: /etc/unbound/unbound.conf.d/dietpi.conf:68: error: unknown keyword 'no'
Nov 06 21:35:09 VM-Stretch unbound[311]: read /etc/unbound/unbound.conf failed: 3 errors in configuration file
Nov 06 21:35:09 VM-Stretch unbound[311]: [1604694909] unbound[311:0] fatal error: Could not read config file: /etc/unbound/unbound.conf
```
Removing it solves the issue.

The changelog is not based on version but on single dates only, however based on the commit date it fits well be have been added after v1.6: https://github.com/NLnetLabs/unbound/commit/9b4b0de74630dd92b79612de4a75e69e1342f58f